example/passthrough_hp.cc

```diff
@@ -1211,6 +1214,13 @@ static cxxopts::ParseResult parse_options(int argc, char **argv) {
     if (fs.debug || fs.debug_fuse)
         fs.foreground = true;
 
+    auto mnt_opts = options["o"].as<std::string>();
+    if (mnt_opts != "" && mnt_opts != "rw") {
+        std::cout << argv[0] << ": Unsupported mount option(s): " << mnt_opts << "\n";
```

Comment:
Hmm, I admit that I don't like that specific testing for "rw" too much. What is if we want to test with options like "sync,nolazytime" - all these flags handled by the vfs?

Reply:
I guess we can extend it later and add a function to iterate over all parameters and to check if these are vfs parameters.